_includes/v20.1/app/start-cockroachdb.md

```diff
@@ -43,7 +43,7 @@ Choose whether to run a temporary local cluster or a free CockroachDB cluster on
 1. On the **Overview** page, click **Create Cluster**.
 1. On the **Create new cluster** page, for **Cloud provider**, select **Google Cloud**.
 1. For **Regions & nodes**, use the default selection of `California (us-west)` region and 1 node.
-1. For **Hardware per node**, select `Option 1` (2vCPU, 60 GB disk).
+1. Under **Hardware per node**, select 2vCPU for **Compute** and a 60 GB disk for **Storage**.
```

Comment:
I would suggest either 35 GiB or 75 GiB here. We don't have 60 GiB in the dropdown option anymore.